homeassistant/components/smartthings/__init__.py

```diff
@@ -194,6 +178,45 @@
     return all(await asyncio.gather(*tasks))
 
 
+async def async_remove_entry(
+        hass: HomeAssistantType, entry: ConfigEntry) -> None:
+    """Perform clean-up when entry is being removed."""
+    from pysmartthings import SmartThings
+
+    api = SmartThings(async_get_clientsession(hass),
+                      entry.data[CONF_ACCESS_TOKEN])
+
+    # Remove the installed_app, which if already removed raises a 403 error.
+    installed_app_id = entry.data[CONF_INSTALLED_APP_ID]
+    try:
+        await api.delete_installed_app(installed_app_id)
+    except ClientResponseError as ex:
+        if ex.status == 403:
+            _LOGGER.exception("Installed app %s has already been removed",
```

Comment:
If this is expected, why log it as exception? I would think debug would be appropriate.

Reply:
Yes , should be `debug`